pagerduty/resource_pagerduty_extension.go

```diff
@@ -174,3 +188,35 @@ func flattenExtensionObjects(serviceList []*pagerduty.ServiceReference) interfac
 	}
 	return services
 }
+
+func expandExtensionConfig(v interface{}) interface{} {
+	var config interface{}
+	if err := json.Unmarshal([]byte(v.(string)), &config); err != nil {
+		log.Printf("[ERROR] Could not unmarshal extension config %s: %v", v.(string), err)
+		return nil
+	}
+
+	return config
+}
+
+func flattenExtensionConfig(config interface{}) interface{} {
```

Comment:
Similar question like the above but for the flatten function: https://github.com/hashicorp/terraform/blob/master/helper/structure/flatten_json.go